process/availability.py

```diff
@@ -1165,8 +1239,19 @@ def cp_lifetime(self):
             cplife = min(ctv.nflutfmax / (fwbsv.neut_flux_cp * YEAR_SECONDS), cv.tlife)
 
         # Aluminium/Copper magnets CP lifetime
-        # For now, we keep the original def, developped for GLIDCOP magnets ...
+        # For now, we keep the original def, developed for GLIDCOP magnets ...
         else:
             cplife = min(cv.cpstflnc / pv.wallmw, cv.tlife)
 
         return cplife
+
+    def divertor_lifetime(self):
```

Comment:
If this isnt handling class attributes then it should not contain self and have the static decorator